routers/web/repo/repo.go

```diff
@@ -579,20 +578,11 @@ func SearchRepo(ctx *context.Context) {
 	}
 
 	// collect the latest commit of each repo
-	repoIDsToLatestCommitSHAs := make(map[int64]string)
-	wg := sync.WaitGroup{}
-	wg.Add(len(repos))
+	// at most there are dozens of repos (limited by MaxResponseItems), so it's not a big problem at the moment
+	repoIDsToLatestCommitSHAs := make(map[int64]string, len(repos))
 	for _, repo := range repos {
-		go func(repo *repo_model.Repository) {
-			defer wg.Done()
-			commitID, err := repo_service.GetBranchCommitID(ctx, repo, repo.DefaultBranch)
-			if err != nil {
-				return
-			}
-			repoIDsToLatestCommitSHAs[repo.ID] = commitID
-		}(repo)
+		repoIDsToLatestCommitSHAs[repo.ID], _ = repo_service.GetBranchCommitID(ctx, repo, repo.DefaultBranch)
```

Comment:
```suggestion
		commitID, err := repo_service.GetBranchCommitID(ctx, repo, repo.DefaultBranch)
		if err != nil {
		  continue
		}
		repoIDsToLatestCommitSHAs[repo.ID] = commitID
```

Reply:
Why? I do not see the necessary.

Reply:
I don't want to pass garbage to `git_model.CalcCommitStatus`